target-platforms/commons.target

```diff
@@ -1,42 +1,70 @@
-      <dependencies>
-        <dependency>
-          <groupId>org.sonarsource.sonarlint.core</groupId>
-          <artifactId>sonarlint-core-osgi</artifactId>
-          <version>8.14.0.62738</version>
-          <type>jar</type>
-        </dependency>
-        <dependency>
-          <groupId>org.jetbrains.kotlin</groupId>
-          <artifactId>kotlin-osgi-bundle</artifactId>
-          <version>1.6.21</version>
-          <type>jar</type>
-        </dependency>
-      </dependencies>
-    </location>
-    <location includeDependencyDepth="infinite" includeDependencyScopes="compile,runtime" includeSource="true" missingManifest="error" type="Maven">
-      <dependencies>
-        <dependency>
-          <groupId>org.assertj</groupId>
-          <artifactId>assertj-core</artifactId>
-          <version>3.24.2</version>
-          <type>jar</type>
-        </dependency>
-        <dependency>
-          <groupId>org.mockito</groupId>
-          <artifactId>mockito-core</artifactId>
-          <version>5.1.1</version>
-          <type>jar</type>
-        </dependency>
-      </dependencies>
-    </location>
+	  <location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" missingManifest="error" type="Maven">
+		  <dependencies>
+			  <dependency>
+				  <groupId>org.jetbrains.kotlin</groupId>
+				  <artifactId>kotlin-osgi-bundle</artifactId>
+				  <version>1.6.21</version>
+				  <type>jar</type>
+			  </dependency>
+			  <dependency>
+				  <groupId>org.sonarsource.sonarlint.core</groupId>
+				  <artifactId>sonarlint-core-osgi</artifactId>
+				  <version>8.14.0.62738</version>
+				  <type>jar</type>
+			  </dependency>
+		  </dependencies>
+	  </location>
+	  <location includeDependencyDepth="infinite" includeDependencyScopes="compile,runtime" includeSource="true" missingManifest="error" type="Maven">
+		  <dependencies>
+			  <dependency>
+				  <groupId>org.assertj</groupId>
+				  <artifactId>assertj-core</artifactId>
+				  <version>3.24.2</version>
+				  <type>jar</type>
+			  </dependency>
+			  <dependency>
+				  <groupId>org.mockito</groupId>
+				  <artifactId>mockito-core</artifactId>
+				  <version>5.1.1</version>
+				  <type>jar</type>
+			  </dependency>
+		  </dependencies>
+	  </location>
+	  <location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" missingManifest="generate" type="Maven">
+		  <dependencies>
+			  <dependency>
+				  <groupId>com.squareup.okhttp3</groupId>
+				  <artifactId>okhttp</artifactId>
+				  <version>4.10.0</version>
+				  <type>jar</type>
+			  </dependency>
+		  </dependencies>
+		  <instructions><![CDATA[
+Bundle-Name:           OkHttp
+version:               ${version_cleanup;${mvnVersion}}
+Bundle-SymbolicName:   com.squareup.okhttp3
+Bundle-Version:        ${version}
+Export-Package:        okhttp3;version="${version}";-noimport:=true
+Import-Package:        okio;version="3.0.0",kotlin.*;version="1.6.20"
```

Comment:
Why is this needed ?

Reply:
Because okhttp is not an OSGi library (while okio is...):
![image](https://user-images.githubusercontent.com/281596/220865379-4a2d8c6a-1829-4920-b61c-816e1171109b.png)

Reply:
Full OSGi support is coming in version 5: https://square.github.io/okhttp/changelogs/changelog/#version-500-alpha1